From cf6a0ce1a526d4ed4b31bef2ee0ccf2a0e72db94 Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Sun, 20 Dec 2020 03:48:32 +0000
Subject: [PATCH 3/4] CPUInfo: Remove the "#X:" prefixes on CPU readings to
 make the output visually easier to read on-screen (fewer numbers to look at).

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 xbmc/utils/CPUInfo.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/xbmc/utils/CPUInfo.cpp b/xbmc/utils/CPUInfo.cpp
index 9f43c1025a..99abb4c388 100644
--- a/xbmc/utils/CPUInfo.cpp
+++ b/xbmc/utils/CPUInfo.cpp
@@ -49,9 +49,9 @@ std::string CCPUInfo::GetCoresUsageString()
         if (!strCores.empty())
           strCores += ' ';
         if (core.m_usagePercent < 10.0)
-          strCores += StringUtils::Format("#{}: {:1.1f}%", core.m_id, core.m_usagePercent);
+          strCores += StringUtils::Format("#{}: {:1.1f}% ", core.m_id, core.m_usagePercent);
         else
-          strCores += StringUtils::Format("#{}: {:3.0f}%", core.m_id, core.m_usagePercent);
+          strCores += StringUtils::Format("#{}: {:3.0f}% ", core.m_id, core.m_usagePercent);
       }
     }
     else
-- 
2.17.1

